modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -129,6 +132,30 @@
  * Command line handler test.
  */
 public class GridCommandHandlerTest extends GridCommandHandlerAbstractTest {
+    /** */
+    private File defaultDiagnosticDir;
+    /** */
+    private File customDiagnosticDir;
+
+    /** {@inheritDoc} */
+    @Override protected void beforeTest() throws Exception {
+        super.beforeTest();
+
+        defaultDiagnosticDir = new File(U.defaultWorkDirectory() + "/" + DEFAULT_TARGET_FOLDER + "/");
+        customDiagnosticDir = new File(U.defaultWorkDirectory() + "/diagnostic_test_dir/");
```

Comment:
I think we could merge field`s definition and field`s initialization to single command.